Assets/MRTK/Providers/OpenVR/HPMotionController.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.OpenVR.Input
+{
+    /// <summary>
+    /// Open VR Implementation of the Windows Mixed Reality Motion Controllers.
+    /// </summary>
+    [MixedRealityController(
+        SupportedControllerType.WindowsMixedReality,
+        new[] { Handedness.Left, Handedness.Right },
+        "Textures/MotionController")]
```

Comment:
Looks like the attribute matches the existing WMR one, but should instead match the attribute on the other HP controllers in this PR